src/components/ProductScreenshot/index.js

```diff
@@ -2,9 +2,26 @@ import React from 'react'
 
 export const ProductScreenshot = ({ imageLight, imageDark, alt, classes }) => {
     return (
-        <div className="mb-4 border border-light dark:border-dark rounded p-4 bg-accent dark:bg-accent-dark">
-            <img src={imageLight} alt={alt} className={`${imageDark ? 'dark:hidden' : ''} ${classes}`} />
-            {imageDark && <img src={imageDark} alt={alt} className={`hidden dark:block ${classes}`} />}
+        <div
+            className={`mb-4 border border-light dark:border-dark rounded p-4 bg-accent dark:bg-accent-dark ${classes}`}
+            style={{ display: 'flex', justifyContent: 'center' }}
```

Comment:
I think we can replace the style with tailwind? Can you add to the PR why this change is necessary, I know you already had it in Slack.
```suggestion
            className={`mb-4 border border-light dark:border-dark rounded p-4 bg-accent dark:bg-accent-dark flex justify-center ${classes}`}
```

Reply:
Sounds good! Updated - thanks!